doc/doxygen/images/quad_refinement_test.svg

```diff
@@ -0,0 +1,1018 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
It seems that also this file is not used anywhere. Can we remove it?

Reply:
This file will be used in the future.